core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -8,22 +8,27 @@
             <div class="form-group" ng-class="{'has-error': !explorationTitleService.displayed}">
               <label for="explorationTitle" class="col-lg-2 col-md-2 col-sm-2">Title</label>
               <div class="col-lg-10 col-md-10 col-sm-10">
-                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40">
+                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40" data-ng-trim="false">
```

Comment:
It allows whitespace before/after the string to count dynamically. Default value is true which results the counting of space only after a character. 
Example: V     H (Here the value of characters are updated after H is typed, i.e 5 to 10)
With data-ng-trim, V    H  (The values are incremented space by space )